### ISSUE TITLE
[Bug]: Typo in seprator

### ISSUE THREAD

**Issue by gijs-martens** [sujaykumarh/docsify-plugin-title#40]
### What happened? Describe the bug

The configuration currently accepts a `seprator` option.
The correct spelling for this would be `separator`.

### Expected behavior

`window.$docsify.customPageTitle.separator` to be recognised

### Your OS

iOS

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by AlexCPU** [comment]
Eeek, this has been 'fixed' as a breaking change on a minor revision.

People using the suggested CDN snippet, which calls up version 1.x:
```
<script src="//cdn.jsdelivr.net/npm/@sujaykumarh/docsify-plugin-title@1.x/dist/plugin.min.js"></script>
```

Will have suddenly discovered that this functionality stopped working on their site, and had to come here to find out why.  This should have been a major version number change, or have continued supporting the incorrect spelling such that it didn't break existing deployments.

**Comment by sujaykumarh** [comment]
Fixed with [v2.0.0](https://github.com/sujaykumarh/docsify-plugin-title/releases/tag/v2.0.0), thanks for bringing it up @AlexCPU. Cheers!
